### PR TITLE
Update end of year post and team assignments

### PR DESCRIPTION
**Pull request by choldgraf**
This PR updates our information to reflect the latest information about our team, and adds information about our new team members from 2024 to the retrospective post. Here are major changes:

- It updates team name and role to reflect new roles we've defined in Q4.
- It updates the bio for a few team members from copy we've had in other materials.
- It adds @aprilmj to our list of team members.
- It adds a short paragraph about three new hires to our retrospective

### CODE REVIEW

File: content/authors/angus-hollands/_index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ social:
 # Organizational groups that you belong to (for People widget)
 #   Set this to `[]` or comment out if you are not using People widget.
 user_groups:
-- Open Engineering Team
+- Product and Services Team
 ---
 
 Angus Hollands is an Open Source Infrastructure at 2i2c. He was previously a post-doctoral researcher in the Computational High Energy Physics group at Princeton University. He has a long-standing history of working collaboratively in open source projects, such as Executable Books, Jupyter, scikit-hep, and Blender. He is motivated by open-source, open-science, and the FAIR principles to build a more accessible, empowering future for scientific research and publication. His scientific background is in nuclear structure, in which he studied a PhD at the University of Birmingham.
```
Comment (aprilmj) [inline content/authors/angus-hollands/_index.md]: Open Source Applications Engineer

File: content/authors/april-johnson/_index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# Username (this should match the folder name and the name on publications)
+authors:
+- "april-johnson"
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position (e.g., Professor of Artificial Intelligence)
+role: People Operations Lead
```
Comment (choldgraf) [inline content/authors/april-johnson/_index.md:13]:
```suggestion
role: Grand poo-bah of People
```

File: content/authors/april-johnson/_index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+- icon: envelope
+  icon_pack: fas
+  link: 'mailto:april@2i2c.org'
+
+
```
Comment (choldgraf) [inline content/authors/april-johnson/_index.md:37]:
```suggestion
# Social/Academic Networking
# Multiple lines
# at once!
social:
- icon: envelope
  icon_pack: fas
  link: 'mailto:april@2i2c.org'


```

File: content/authors/jenny-wong/_index.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ social:
 # Organizational groups that you belong to (for People widget)
 #   Set this to `[]` or comment out if you are not using People widget.
 user_groups:
-- Partnerships Team
+- Product and Services Team
 ---
 
 Jenny is the Technical Content Developer for 2i2c. Her work is mainly focused on the CZI-funded Catalyst Project, an initiative aiming to provide research communities in Latin America and Africa with access to large-scale scientific infrastructure. As part of this effort, she will help develop community-based training materials for interactive cloud-native workflows.
```
Comment (aprilmj) [inline content/authors/jenny-wong/_index.md]:
```suggestion
Jenny is a Product Manager focused on Platform and Services for 2i2c. Her work has included the CZI-funded Catalyst Project, an initiative aiming to provide research communities in Latin America and Africa with access to large-scale scientific infrastructure. She's built community-based training materials for interactive cloud-native workflows. Jenny is responsible for direct community engagement to ensure the alignment of Product and Services value streams to the needs of the communities we serve. 
```

File: content/authors/james-munroe/_index.md [hunk 5]
```diff
@@ -44,7 +44,7 @@ social:
 # Organizational groups that you belong to (for People widget)
 #   Set this to `[]` or comment out if you are not using People widget.
 user_groups:
-- Partnerships Team
+- Product and Services Team
 ---
 
 James is Product and Community Lead for 2i2c. Coming from a background as an associate professor in the Department of Physics and Physical Oceanography at [Memorial University of Newfoundland](https://mun.ca), he is a strong advocate for enabling scientists and students to be efficient and effective in their computational workflows. Building on previous work in big data oceanography with links to the [Pangeo](https://pangeo.io/) project, [COSIMA](http://cosima.org.au/): Consortium for Ocean-Sea Ice Modelling in Australia, and [CIOOS](https://cioos.ca/): Canadian Integrated Ocean Observing System, James wants to bring the strength of the Jupyter ecosystem to users across a broad range of educational and research domains.
```
Comment (aprilmj) [inline content/authors/james-munroe/_index.md]:
```suggestion
James is Senior Product Manager, Open Science Enablement for 2i2c. Coming from a background as an associate professor in the Department of Physics and Physical Oceanography at [Memorial University of Newfoundland](https://mun.ca), he is a strong advocate for enabling scientists and students to be efficient and effective in their computational workflows. Building on previous work in big data oceanography with links to the [Pangeo](https://pangeo.io/) project, [COSIMA](http://cosima.org.au/): Consortium for Ocean-Sea Ice Modelling in Australia, and [CIOOS](https://cioos.ca/): Canadian Integrated Ocean Observing System, James wants to bring the strength of the Jupyter ecosystem to users across a broad range of educational and research domains.
```

File: content/authors/april-johnson/_index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Delivery Enablement Team
+---
+
+April brings ideas to life, whether that's making big change happen or creating teams from scratch. She has helped people make their businesses greener, design and build their technology and talent organizations, go public, incorporate new capabilities through acquisition, and scale up for the future. (from April's [LinkedIn profile](https://www.linkedin.com/in/apriljohnson/))
```
Comment (aprilmj) [inline content/authors/april-johnson/_index.md]:
```suggestion
April is a people & transformation consultant who brings ideas to life, whether that's making big change happen or creating teams from scratch. She has helped people make their businesses greener, design and build their technology and talent organizations, go public, incorporate new capabilities through acquisition, and scale up for the future. 
```

**Review by aprilmj — CHANGES_REQUESTED** [review]
Make sure to update titles where they'd existed in bio text - I think I got everyone's

File: content/authors/angus-hollands/_index.md (hunk 1 above)
Comment (choldgraf) [inline content/authors/angus-hollands/_index.md]:
```suggestion
Angus Hollands is an Open Source Applications Engineer at 2i2c. He was previously a post-doctoral researcher in the Computational High Energy Physics group at Princeton University. He has a long-standing history of working collaboratively in open source projects, such as Executable Books, Jupyter, scikit-hep, and Blender. He is motivated by open-source, open-science, and the FAIR principles to build a more accessible, empowering future for scientific research and publication. His scientific background is in nuclear structure, in which he studied a PhD at the University of Birmingham.
```

**Comment by choldgraf** [comment]
OK merging this one since we had a few rounds of feedback already, LJ also confirmed she liked the changes. Thanks @aprilmj for the ideas!

**Pull request merged**